IDE/AURIX/test-app/Lcf_Gnuc_Tricore_Tc.lsl

```diff
@@ -93,37 +101,40 @@ MEMORY
     dsram2_local (w!xp): org = 0xd0000000, len = 96K
     dsram2 (w!xp): org = 0x50000000, len = 96K
     psram2 (w!xp): org = 0x50100000, len = 64K
-    
+
     dsram1_local (w!xp): org = 0xd0000000, len = 240K
     dsram1 (w!xp): org = 0x60000000, len = 240K
     psram1 (w!xp): org = 0x60100000, len = 64K
-    
+
     dsram0_local (w!xp): org = 0xd0000000, len = 240K
     dsram0 (w!xp): org = 0x70000000, len = 240K
     psram0 (w!xp): org = 0x70100000, len = 64K
-    
+
     psram_local (w!xp): org = 0xc0000000, len = 64K
-    
+
     /* Majority of pfls0 is reserved for wolfBoot, swap and HSM */
-    pfls0 (rx!p):    org = 0x80000000, len = 3M
-    
+    pfls0 (rx!p):    org = 0xA0000000, len = 3M
```

Comment:
It is required to make a contiguous binary when we generate a bin file for wolfBoot to sign. Otherwise the binary generated would be massive since there would be MBs of holes between load VMAs. It has no effect on the underlying code or symbols exported.